lang/python/pillow/Makefile

```diff
@@ -7,16 +7,17 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=pillow
-PKG_VERSION:=5.4.1
+PKG_VERSION:=6.0.0
```

Comment:
nitpick: this changeset does 2 upgrades;
if possible, just do one to 6.0.0 directly;

Reply:
Ok, done.